lib/call-cabal-project-to-nix.nix

```
@@ -246,6 +247,27 @@ let
       repoResult = pkgs.haskell-nix.haskellLib.parseRepositories
         evalPackages cabalProjectFileName sha256map inputMap cabal-install nix-tools sourceRepoPackageResult.otherText;
 
+      extraSourceRepo = source:
+        let
+          src =
+            pkgs.evalPackages.runCommand "extra-source-repository-package" {
+              nativeBuildInputs = [ pkgs.evalPackages.rsync pkgs.evalPackages.git ];
+            } ''
```

I get `attribute 'evalPackages' missing`; given that we pass `evalPackages` to this; should it actually be 
```suggestion
            evalPackages.runCommand "extra-source-repository-package" {
              nativeBuildInputs = [ evalPackages.rsync evalPackages.git ];
            } ''
```